### PR TITLE
add DSEG

### PR DESCRIPTION
**Pull request by master-hax**
add DSEG font (https://github.com/keshikan/DSEG)

### CODE REVIEW

**Comment by master-hax** [comment]
@googlebot I signed it!

**Comment by master-hax** [comment]
@googlebot I fixed it.

**Comment by master-hax** [comment]
@googlebot I fixed it.

**Comment by master-hax** [comment]
do we add to [CONTRIBUTORS](https://github.com/google/fonts/blob/master/CONTRIBUTORS) or [AUTHORS](https://github.com/google/fonts/blob/master/AUTHORS)? looks like they have not been  touched in a while

File: ofl/DSEG/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Copyright (c) 2020, keshikan (http://www.keshikan.net),
+with Reserved Font Name "DSEG"
```
Comment (davelab6) [inline ofl/DSEG/OFL.txt:2]: We prefer not to accept fonts with RFNs.

**Comment by davelab6** [comment]
I am open to onboarding 1 or possibly more of the fonts in this project, but we require one folder per family name, and so on per our [Spec](https://github.com/googlefonts/gf-docs/tree/master/Spec) and other [CONTRIBUTING](https://github.com/google/fonts/blob/master/CONTRIBUTING.md) requirements. 

The glyph set is unclear to me, it would be good to see how the glyph set of each family maps to the GF Latin Core set.

**Comment by RosaWagner** [comment]
Since we won't merge this PR, I am closing it. I invite @master-hax to open an issue to discuss the project and the necessary conditions to onboard one or several fonts of this project.

**Pull request closed without merging**